### DISCUSSION TITLE
Why you prevent Copilot from solving math problems?

### DISCUSSION THREAD

**Discussion by wenh06** [community/community#102286] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Until a few months ago, Copilot was able to solve math problems:
![4367bb739d5192c691cae5541f62dd1](https://github.com/community/community/assets/8778305/84047bec-2694-44f3-a014-fc6f866cb897)
(the above answer is almost correct, but needs modification)
But now it just repeats:
![image](https://github.com/community/community/assets/8778305/1103c03b-78b9-4f65-8b00-a27627cbc83b)

**Comment by mw-xyz** [comment]
I tried the same with a physics question. I think it's because GPT-4 in general isn't the best at solving maths problems and they don't want it to pass over into copilot, which you have to pay for in general, and would prefer if you went somewhere else as it is meant to be like a great product. Hope this makes sense and helped you :)
